pages/services/edge-lb/1.4/tutorials/customizing-templates/index.md

```diff
@@ -0,0 +1,144 @@
+
+# Before you begin
+* You must have an active and properly-configured DC/OS Enterprise cluster.
+* You must have Edge-LB installed as described in the Edge-LB [installation instructions](/services/edge-lb/1.4/getting-started/installing).
+* You must have the `edgelb` command-line interface (CLI) installed.
+
+# Create a custom template
+
+To create a custom template for Basic authentication:
+
+1. Install Edge-LB as described [Installing Edge-LB](/services/edge-lb/1.4/getting-started/installing).
+
+1. Create an Edge-LB pool as described in [Expose and load balance a service](/services/edge-lb/1.4/getting-started/single-lb).
+
+1. Fetch the default template for the Edge-LB pool and save it to a file names `haproxy.tmpl` by running the following command:
+
+    ```bash
+    dcos edgelb template show > haproxy.tmpl
+    ```
+
+1. Open the `haproxy.tmpl` template you created in the previous step in a text editor.
+
+1. Modify the `haproxy.tmpl` file to create a `basic-auth` user list with user names and passwords for a set of regular and administrative users.
+
+    For example:
+
+    ```text
+    userlist basic-auth
+      group regular-users
+      group admin-users
+
+      user admin  password ASYRtiLFCipT6 groups admin-users
+      user drew password n5se4LwUfAW1sqA groups regular-users
+      user luz password x18R29iAdV/$1QU groups regular-users
+      user guest insecure-password guestpassword
+    ```
+
+1. Create an access control list (ACL) rule inside of the `backend` section for the users and groups you defined in the `userlist` who are allowed to access the load balanced service.
+
+    For example:
+
+    ```text
+    acl example-auth http_auth(basic-auth)
+    http-request auth realm example unless example-auth
+    ```
+
+1. Create an access control list (ACL) rule inside `backend` section to grant different access rights to users who belong to the `admin-users` group  that you defined in the `userlist`.
+
+    For example:
+
+    ```text
+    acl itadmin-auth http_auth_group(basic-auth) admin-users
+    http-request auth realm itadmin unless itadmin-auth
+    ```
+
+1. Create additional access control list (ACL) rules inside `frontend` section to grant limited access to the users who do not belong to the `admin-users` or `regular-users` groups that you defined in the `userlist`.
+
+    For example:
+
+    ```text
+    acl guest-users hdr_dom(host) -i guest.example.org
+    acl basic-auth-user http_auth(basic-auth)
+    acl basic-auth-user-with-group  http_auth_group(basic-auth) admin-users regular-users
+    http-request auth realm guest if guest-users !basic-auth-user OR guest-users basic-auth-user-with-group
+    use_backend web-guest-production if guest-users
+    ```
+
+1. Verify your user list and the access control rules you have set for the `frontend` and `backend` sections.
+
+For example:
+
+    ```text
```

Comment:
For some reason, `text` is coming while rendering this file.

Reply:
changed to bash